eva/optimizer/operators.py

```diff
@@ -547,6 +547,7 @@ class LogicalCreate(Operator):
         video {TableRef}: [video table that is to be created]
         column_list {List[ColumnDefinition]}:
         if_not_exists {bool}: [create table if exists]
+        query {SelectStatement}: Optional, only if new table is created by a query output
```

Comment:
Shall we remove it, since the operator does not have the query as a member variable?

Reply:
agree. this can be deleted